planetiler-core/src/main/java/com/onthegomap/planetiler/Planetiler.java

```diff
@@ -616,7 +619,7 @@ public void run() throws Exception {
     stats.monitorFile("nodes", nodeDbPath);
     stats.monitorFile("features", featureDbPath);
     stats.monitorFile("multipolygons", multipolygonPath);
-    stats.monitorFile("mbtiles", output);
+    stats.monitorFile("archive", output);
```

Comment:
I'm thinking we might want a third file output that just writes files to disk. Do you think "archive" is appropriate for that output as well? If not we might change to "output"

Reply:
Yeah, I guess technically `TileArchive` is a misnomer for directory of tiles on disk but I can live with it... `output` and `tileset` felt too generic to me: e.g.: two tile archives in MBTiles and PMTiles format represent the same output tileset